dist/tools/riot_hdr/main.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2017 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU General Public
+ * License v2. See the file LICENSE for more details.
+ */
+
+/**
+ * @file
+ * @brief       Header generation tool for RIOT firmware images
+ *
+ * @author      Francisco Acosta <francisco.acosta@inria.fr>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+int genhdr(int argc, char *argv[]);
+extern const char generate_usage[];
```

Comment:
If these are shared they should go in a header instead of hacking this here without function documentation.

Reply:
`genhdr` is only used by main if the command `generate` is used.
`generate_usage` is only used in `genhdr`.

They're not shared but I can consider moving them to a .h, I just really need to see the befits.